driver-pulsar/deploy/deploy.yaml

```diff
@@ -74,7 +74,11 @@
     - template:
         src: "templates/pulsar_env.sh"
         dest: "/opt/pulsar/conf/pulsar_env.sh"
-
+    - name: Change locale to en_US.utf-8
```

Comment:
Probably we'd need the same in `driver-kafka/deploy/deploy.yaml`

Reply:
Thanks, changed it.